libs/designer-ui/src/lib/arrayeditor/index.tsx

```diff
@@ -34,6 +34,7 @@ export interface ArrayItemSchema {
   format?: string;
   enum?: string[];
   readOnly?: boolean;
+  ['x-ms-visibility']: string;
```

Comment:
PLease use the swagger constant instead of coding here, this is to make sure extension names are used from single place. check schemaprocessor